src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx

```diff
@@ -508,7 +508,12 @@ export default function AppMarketCard() {
               }}
               name="title"
               pattern={Patterns.appMarketCard.appTitle}
-              label={t('content.apprelease.appMarketCard.appTitle') + ' *'}
+              label={
+                <>
+                  {t('content.apprelease.appMarketCard.appTitle')}
+                  <span style={{ color: 'red' }}> *</span>
+                </>
```

Comment:
in locale file -> "serviceName": "Service Name<0>*</0>",

the final typography looks something like this
```
<Typography>

<Trans
              i18nKey={label}
              components={[
<span style={{color: 'red'}}></span>,
              ]}
></Trans>

</Typography>
```